lib/index.ts

```diff
@@ -23,15 +23,9 @@ export const useStoryblokBridge = <
   cb: (newStory: ISbStoryData<T>) => void,
   options: StoryblokBridgeConfigV2 = {}
 ) => {
-  if (typeof window === "undefined") {
-    return;
-  }
-
-  if (typeof window.storyblokRegisterEvent === "undefined") {
-    console.error(
-      "Storyblok Bridge is disabled. Please enable it to use it. Read https://github.com/storyblok/storyblok-js"
-    );
-
+  const isServer = typeof window === "undefined";
+  const isBridgeLoaded = typeof window.storyblokRegisterEvent !== "undefined";
```

Comment:
I think that `typeof window.storyblokRegisterEvent` will throw an error if `window` is undefined. You could check if the typeof window is not undefined before checking this property

Reply:
great catch! totally missed using previous boolean here